OpenOversight/migrations/versions/19f23f6eb170_.py

```diff
@@ -0,0 +1,21 @@
+"""empty message
+
+Revision ID: 19f23f6eb170
+Revises: af933dc1ef93
+Create Date: 2018-05-06 12:55:02.139610
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '19f23f6eb170'
+down_revision = 'af933dc1ef93'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    pass
```

Comment:
As you can see, your `upgrade()` and `downgrade()` does nothing here, because Alembic [does not autogenerate migrations for column renames](http://alembic.zzzcomputing.com/en/latest/autogenerate.html#what-does-autogenerate-detect-and-what-does-it-not-detect). Unfortunately, that means that in order to merge the changes in this PR, we need a manually written migration - note that we cannot simply drop the old column and add the new column, as then we'd lose all the data in the column.

Reply:
```
def upgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    op.alter_column('assignments', 'start_no',
                    existing_type=sa.INTEGER(),
                    type_=sa.String(length=120),
                    existing_nullable=True)
    # ### end Alembic commands ###
```

```
def downgrade():
     # ### commands auto generated by Alembic - please adjust! ###
    op.alter_column('assignments', 'start_no',
                    existing_type=sa.String(length=120),
                    type_=sa.INTEGER(),
                    existing_nullable=True)
```

So is this manually written migration enough. Is this what you meant? @redshiftzero @raq929